src/main/java/frc/robot/Robot.java

```diff
@@ -142,24 +142,25 @@ public void configureButtonBindings() {
                 }));
 
         driveController.y().onTrue(led.awesome());
-        driveController.rightBumper().onTrue(commandSequences.chargeAmp());
         driveController.a().onTrue(drive.rotateToSpeakerTarget());
 
-        copilotController.back().onTrue(intake.safeStateCmd());
+        copilotController.back().onTrue(intake.safeStateCmd().andThen(armRotate.safeStateCmd()));
         copilotController.start().onTrue(shooter.setSpeed(Speeds.OFF));
         copilotController.a().onTrue(commandSequences.moveAndIntake());
         // copilotController.b().onTrue(undertaker.safeStateCmd());
-        copilotController.b().whileTrue(commandSequences.scoreSpeakerCharge(copilotController, driveController));
-        copilotController.b().onFalse(commandSequences.scoreSpeakerRelease(copilotController, driveController));
-        copilotController.y().onTrue(commandSequences.scoreSpeakerCenterlineCharge(copilotController, driveController));
+        copilotController.b()
+                .whileTrue(commandSequences.charge(Speeds.SUBWOOFER_SHOT, ArmPresets.SCORE_SPEAKER_SUBWOOFER));
+        copilotController.b().onFalse(commandSequences.release());
```

Comment:
```suggestion
        copilotController.b()
                .whileTrue(commandSequences.charge(Speeds.SUBWOOFER_SHOT, ArmPresets.SCORE_SPEAKER_SUBWOOFER)).onFalse(commandSequences.release());
```
These functions can be chained together instead of getting the button again.

Reply:
And it looks like we're using `onTrue` instead of `whileTrue` for some of these below.